airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -133,11 +144,33 @@ protected JdbcSqlGenerator getSqlGenerator() {
     throw new UnsupportedOperationException("mysql does not yet support DV2");
   }
 
+  @Override
+  protected StreamAwareDataTransformer getDataTransformer(ParsedCatalog parsedCatalog, String defaultNamespace) {
```

Comment:
see MySQLSqlOperations - this is replacing the `formatData` override. (@gisripa doublechecking - am I understanding these two methods correctly?)

Reply:
Yes correct. your impl of StreamAwareDataTransformer will make it consistent across all.

Reply:
oh nice, can I just kill formatData then?

Reply:
doesn't seem like there's any overrides, I'll remove it
![image](https://github.com/airbytehq/airbyte/assets/5741425/94a1f78c-93b0-456d-9e37-84b2c2563c0a)

Reply:
https://github.com/airbytehq/airbyte/pull/36926/commits/2b3e0f548da1f487db7a16c30b4d477923793954